### PR TITLE
feat: add prettier and eslint (#18)

### PR DESCRIPTION
**Pull request by MC-kanon**
feat: add prettier and eslint #18

### CODE REVIEW

File: web/.prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    printWidth: 80, // 每行代码长度（默认80）
```
Comment (cubxxw) [inline web/.prettierrc.js:2]: Maybe we should use English for the comments

**Review by cubxxw — COMMENTED** [review]
lgtm

**Comment by IRONICBo** [comment]
lgtm

File: web/index.html [hunk 2]
```diff
@@ -16,15 +16,15 @@
 -->
```
Comment (AllianceTing) [inline web/index.html:16]: no ‘-->’

File: web/package.json [hunk 3]
```diff
@@ -1,23 +1,29 @@
...
-  "scripts": {
-    "dev": "vite",
-    "build": "vite build",
-    "preview": "vite preview"
-  },
-  "dependencies": {
-    "@element-plus/icons-vue": "^2.1.0",
-    "axios": "^1.4.0",
-    "echarts": "^5.4.2",
-    "element-plus": "^2.3.7",
```
Comment (AllianceTing) [inline web/package.json:14]: suggest to use acro-design or Tdesign
element-plus is up to date

**Review by AllianceTing — COMMENTED** [review]
approve!

**Comment by MC-kanon** [comment]
Thanks for CR.

**Review by cubxxw — APPROVED** [review]
lgtm

**Comment by IRONICBo** [comment]
lgtm

File: web/package.json (hunk 3 above)
Comment (IRONICBo) [reply web/package.json:14]: Tdesign looks better.

**Review by IRONICBo — APPROVED** [review]

**Pull request merged**